app/forms/collection_form.rb

```diff
@@ -37,9 +37,9 @@ def sync
 
   # @return [Dor::Collection] registers the Collection
   def register_model
-    response = Dor::RegistrationService.create_from_request(register_params)
+    response = DorServices::Client.register(params: register_params)
     # Once it's been created we populate it with its metadata
-    @model = Dor.find(response[:pid])
+    Dor.find(response[:pid])
```

Comment:
Why do we need to find this?

Reply:
So that it is assigned to `@model` in the calling method (which the form uses throughout). I moved the assignment out of this method to make it easier to test.

Reply:
Ah, I see now! 👍